chrome/content/zoteroocr.js

```diff
@@ -5,16 +5,29 @@ Components.utils.import("resource://gre/modules/FileUtils.jsm");
 Zotero.OCR = new function() {
 
 	this.recognize = Zotero.Promise.coroutine(function* () {
-		var ocrEngine = Zotero.Prefs.get("zoteroocr.ocrPath") || "C:\\Program Files\\Tesseract-OCR\\tesseract.exe";
-		alert(ocrEngine);
-		// TODO analyze the installed languages and scripts
-		var items = Zotero.getActiveZoteroPane().getSelectedItems();
 		let dir = FileUtils.getDir('CurProcD', []);
 		let pdfinfo = dir.clone();
-		pdfinfo.append("pdfinfo.exe");
 		let pdftoppm = dir.clone();
-		pdftoppm.append("pdftoppm.exe");
 		
+		if (Zotero.isWin) {
+			pdfinfo.append("pdfinfo.exe");
+			pdftoppm.append("pdftoppm.exe");
+			var ocrEngine_default = "C:\\Program Files\\Tesseract-OCR\\tesseract.exe";
+			var png_prefix = "\\page";
+			var imageListString_prefix = "\\page-";
+			var imageList_file = "\\image-list.txt";
+		} else {
+			pdfinfo.append("pdfinfo");
+			pdftoppm.append("pdftoppm");
+			var ocrEngine_default = "tesseract";
```

Comment:
```suggestion
			var ocrEngine_default = "/usr/bin/tesseract";
```

Reply:
Executing commands with `Zotero.Utilities.Internal.exec` requires an absolute path on macOS. I think on Linux the path is needed, too.